epochStart/bootstrap/process.go

```diff
@@ -285,6 +268,21 @@ func (e *epochStartBootstrap) Bootstrap() (Parameters, error) {
 	return e.requestAndProcessing()
 }
 
+func (e *epochStartBootstrap) computeIfCurrentEpochIsSaved() bool {
+	e.initializeFromLocalStorage()
+	if !e.baseData.storageExists {
+		return false
+	}
+
+	computedRound := e.rounder.Index()
+	if computedRound-e.baseData.lastRound < roundGracePeriod {
+		return true
+	}
+
+	roundsSinceEpochStart := computedRound - int64(e.baseData.epochStartRound)
+	return float64(roundsSinceEpochStart) < float64(e.generalConfig.EpochStartConfig.RoundsPerEpoch)*gracePeriodInPercentage
```

Comment:
We don't need to use here MinRoundsBetweenEpochs from config.toml [EpochStartConfig] section instead
float64(e.generalConfig.EpochStartConfig.RoundsPerEpoch)*gracePeriodInPercentage ?

Reply:
MinRoundsBetweenEpoch is used only for forceEndOfEpoch.

Reply:
Yes I know, and I just thinking to this case, just to work even in this situation. But anyway, you can leave it like this right now, if this is not a good suggestion

Reply:
t is impossible to know at start in epoch if there was a force start in epoch or not.